assets/app/view/logo.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/LineLength
+
+require 'lib/color'
+
+module View
+  class Logo < Snabberb::Component
+    needs :user, default: nil, store: true
+
+    def render
+      props = {
+        style: {
+          position: 'absolute',
+          top: '10px',
+          width: '80px',
+          height: '32px',
+        },
+        attrs: {
+          width: '500px',
+          height: '198px',
+          viewBox: '0 0 500 198',
+        },
+      }
+
+      bg_color = @user&.dig(:settings, :bg_color) || '#ffffff'
+      logo_id = bg_color == '#ffffff' ? 'logo--dark' : 'logo'
+      # green = '#71bf44'
+      offboard = '#ec232a'
```

Comment:
you can make these constants in the class

Reply:
Will do and remove #